### ISSUE TITLE
Joins for Subject Map

### ISSUE THREAD

**Issue by DylanVanAssche** [kg-construct/rml-fno-spec#44]
Add support for joins for Subject Maps as well, useful to construct IRIs from multiple tables or CSVs, especially for GTFS files.
Currently, joins are only possible in the Predicate Object Maps.

**Comment by dachafra** [comment]
This will be part of the RML+Joins discussion I think, which is something we haven't started yet ;-) 
Also, I don't know if RML+Joins will be an independent spec or part of the core one

**Comment by DylanVanAssche** [comment]
Yeah I wasn't really sure where to put it, but I was just hit for an use case so I thought, better document it :) 
We can move issues to other repos/specs in Github.

**Comment by dachafra** [comment]
Great!! I'll add it to the roadmap document I'm generating for Monday's meeting

**Comment by pmaria** [comment]
To check/clarify: This is about a single subject , or more generally term, generating function using multiple "joined" sources, right?

**Comment by DylanVanAssche** [comment]
> To check/clarify: This is about a single subject , or more generally term, generating function using multiple "joined" sources, right?

Yes, for example:

- a.txt --> `id1`
- b.txt --> `id2`

with the template: `"http:/example.org/{id1}-{id2}"`

**Comment by dachafra** [comment]
As it's a join issue, I'm going to move it to its corresponding repo
